.packit.yaml

```diff
@@ -0,0 +1,17 @@
+specfile_path: colin.spec
+synced_files:
+  - .packit.json
```

Comment:
Did you mean `.packit.yaml`?

Reply:
and why do we need to sync it?

Reply:
So you can more easily run packit from distgit? (This would need some work on the packit site to make it usable.)

Reply:
good catch

Reply:
but since packit can't do it yet, we shouldn't add it; I actually have one more suggestion, will create a PR instead